### ISSUE TITLE
Add "framework: >= net461" etc. to paket.dependencies where possible

### ISSUE THREAD

**Issue by dsyme** [fsharp-editing/Forge#72]
If forge added

    framework: >= net461

to top of the generated paket.dependencies for the .NET Framework templates then all the Paket resolution steps would be way faster. After adding this we get:

```
C:\GitHub\misc\t1>.paket\paket.exe update
Paket version 5.4.0
Resolving packages for group Main:
 - FAKE 4.62.1
 - FSharp.Core 4.2.1
 - FSharp.NET.Sdk 1.0.5
 - FSharp.Compiler.Tools 4.1.17
Locked version resolution written to C:\GitHub\misc\t1\paket.lock
Garbage collecting Microsoft.NETCore.Platforms
Garbage collecting Microsoft.NETCore.Targets
Garbage collecting runtime.debian.8-x64.runtime.native.System.Security.Cryptography.OpenSsl
Garbage collecting runtime.fedora.23-x64.runtime.native.System.Security.Cryptography.OpenSsl
Garbage collecting runtime.fedora.24-x64.runtime.native.System.Security.Cryptography.OpenSsl
Garbage collecting runtime.native.System
Garbage collecting runtime.native.System.Net.Http
Garbage collecting runtime.native.System.Security.Cryptography.Apple
Garbage collecting runtime.native.System.Security.Cryptography.OpenSsl
Garbage collecting runtime.opensuse.13.2-x64.runtime.native.System.Security.Cryptography.OpenSsl
Garbage collecting runtime.opensuse.42.1-x64.runtime.native.System.Security.Cryptography.OpenSsl
Garbage collecting runtime.osx.10.10-x64.runtime.native.System.Security.Cryptography.Apple
Garbage collecting runtime.osx.10.10-x64.runtime.native.System.Security.Cryptography.OpenSsl
Garbage collecting runtime.rhel.7-x64.runtime.native.System.Security.Cryptography.OpenSsl
Garbage collecting runtime.ubuntu.14.04-x64.runtime.native.System.Security.Cryptography.OpenSsl
Garbage collecting runtime.ubuntu.16.04-x64.runtime.native.System.Security.Cryptography.OpenSsl
Garbage collecting runtime.ubuntu.16.10-x64.runtime.native.System.Security.Cryptography.OpenSsl
Garbage collecting System.Collections
Garbage collecting System.Collections.Concurrent
Garbage collecting System.Console
Garbage collecting System.Diagnostics.Debug
Garbage collecting System.Diagnostics.DiagnosticSource
Garbage collecting System.Diagnostics.Tools
Garbage collecting System.Diagnostics.Tracing
Garbage collecting System.Globalization
Garbage collecting System.Globalization.Calendars
Garbage collecting System.Globalization.Extensions
Garbage collecting System.IO
Garbage collecting System.IO.FileSystem
Garbage collecting System.IO.FileSystem.Primitives
Garbage collecting System.Linq
Garbage collecting System.Linq.Expressions
Garbage collecting System.Linq.Queryable
Garbage collecting System.Net.Http
Garbage collecting System.Net.Primitives
Garbage collecting System.Net.Requests
Garbage collecting System.Net.WebHeaderCollection
Garbage collecting System.ObjectModel
Garbage collecting System.Reflection
Garbage collecting System.Reflection.Emit
Garbage collecting System.Reflection.Emit.ILGeneration
Garbage collecting System.Reflection.Emit.Lightweight
Garbage collecting System.Reflection.Extensions
Garbage collecting System.Reflection.Primitives
Garbage collecting System.Reflection.TypeExtensions
Garbage collecting System.Resources.ResourceManager
Garbage collecting System.Runtime
Garbage collecting System.Runtime.Extensions
Garbage collecting System.Runtime.Handles
Garbage collecting System.Runtime.InteropServices
Garbage collecting System.Runtime.Numerics
Garbage collecting System.Security.Cryptography.Algorithms
Garbage collecting System.Security.Cryptography.Cng
Garbage collecting System.Security.Cryptography.Csp
Garbage collecting System.Security.Cryptography.Encoding
Garbage collecting System.Security.Cryptography.OpenSsl
Garbage collecting System.Security.Cryptography.Primitives
Garbage collecting System.Security.Cryptography.X509Certificates
Garbage collecting System.Text.Encoding
Garbage collecting System.Text.RegularExpressions
Garbage collecting System.Threading
Garbage collecting System.Threading.Tasks
Garbage collecting System.Threading.Tasks.Parallel
Garbage collecting System.Threading.Thread
Garbage collecting System.Threading.ThreadPool
Garbage collecting System.Threading.Timer
Performance:
 - Resolver: 2 seconds (1 runs)
    - Runtime: 180 milliseconds
    - Blocked (retrieving package details): 110 milliseconds (4 times)
    - Blocked (retrieving package versions): 2 seconds (2 times)
    - Not Blocked (retrieving package versions): 2 times
 - Disk IO: 53 milliseconds
 - Average Request Time: 1 second
 - Number of Requests: 12
 - Runtime: 6 seconds
```

then paket update gives just:
```
C:\GitHub\misc\t1>.paket\paket.exe update
Paket version 5.4.0
Resolving packages for group Main:
 - FAKE 4.62.1
 - FSharp.Core 4.2.1
 - FSharp.NET.Sdk 1.0.5
 - FSharp.Compiler.Tools 4.1.17
C:\GitHub\misc\t1\paket.lock is already up-to-date
Performance:
 - Resolver: 1 second (1 runs)
    - Runtime: 195 milliseconds
    - Blocked (retrieving package details): 112 milliseconds (4 times)
    - Blocked (retrieving package versions): 1 second (3 times)
    - Not Blocked (retrieving package versions): 1 times
 - Disk IO: 49 milliseconds
 - Average Request Time: 639 milliseconds
 - Number of Requests: 11
 - Runtime: 3 seconds
```

**Comment by Krzysztof-Cieslak** [comment]
Sounds good. I'll add framework restriction in case where Forge generates `paket.dependency` file, not in case where Forge is used on existing solution with Paket.

**Comment by Krzysztof-Cieslak** [comment]
Added in https://github.com/fsharp-editing/Forge/pull/71/commits/b090cc84ed75650a9c5a55a9927b69566f94ff24, will be available in next 2.0 preview.
